### ISSUE TITLE
Bug in check-consul-servers

### ISSUE THREAD

**Issue by madAndroid** [sensu-plugins/sensu-plugins-consul#4]
When I run `check-consul-servers.rb` with: `check-consul-servers.rb -e 3 -g 3`

I get:
```
Check failed to run: comparison of Fixnum with String failed, ["/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-consul-0.0.5/bin/check-consul-servers.rb:66:in `<='", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-consul-0.0.5/bin/check-consul-servers.rb:66:in `run'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]
```

I'll create a pull-request with a fix I've been using.

**Comment by analytically** [comment]
Merged your PR, released 0.0.6. Enjoy!

**Comment by madAndroid** [comment]
Awesome, thanks! :)
